File: menyaring_data.js
const people = [
  { id: 1, name: "Udin", age: 12 },
  { id: 2, name: "Wati", age: 51 },
  { id: 3, name: "Budi", age: 34 },
  { id: 4, name: "Agus", age: 16 },
  { id: 5, name: "Sari", age: 19 },
  { id: 6, name: "Ririn", age: 21 },
];

var saring = [];
people.forEach(function (sar) {
  if (sar.age < 21) {
    saring.push(sar);
  }
});

console.log(saring);

File: menyaringdata.go
package main

import "fmt"

type People struct {
	id   int
	name string
	age  int
}

func main() {
	people := []People{
		{1, "Udin", 12},
		{2, "Wati", 51},
		{3, "Budi", 34},
		{4, "Agus", 16},
		{5, "Sari", 19},
		{6, "Ririn", 21},
	}

	var mapData []People
	for _, v := range people {
		if v.age < 21 {
			mapData = append(mapData, v)
		}
	}
	fmt.Print(mapData)
}
